Justfile

```diff
@@ -38,15 +38,23 @@ build-environment environment="all":
     cp -rf --no-preserve=mode,ownership "$srcDir"/. {{process-compose-artifacts-dir}}
   else
     destDir="{{process-compose-artifacts-dir}}/{{environment}}"
-    srcDir=$(nix build --impure -L --json .#legacyPackages.{{system}}.process-compose-environments.{{environment}} | jq -r '.[0].outputs.out')
+    if [[ "${USE_LOCAL_CARGO_RESULTS:-0}" = 1 ]]; then
```

Comment:
Could we abstract the only difference between these if branches (the `process-compose-environments{,-with-cargo-local}` into a variable (either a shell or Justfile one) and have only one `srcDir=...` line then